java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -46,6 +46,7 @@ public class TransactionTestUtilities {
     private static final String field1 = UUID.randomUUID().toString();
     private static final String field2 = UUID.randomUUID().toString();
     private static final String field3 = UUID.randomUUID().toString();
+    private static final String key10 = "{key}" + UUID.randomUUID();
```

Comment:
we don't need to create a new key

Reply:
@acarbonetto when i tried using an existing key, it didn't give me the output I wanted so I made a new key.

Reply:
for transactions, you only need to include a very basic test case.  

can you try my suggestion below and see if the tests pass?  You could even run append with an empty string (which shouldn't change the result) and that would be good enough ;)

Reply:
yup will do